src/common/transformations/include/transformations/fix_rt_info.hpp

```diff
@@ -0,0 +1,44 @@
+// Copyright (C) 2018-2021 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+
+/**
+ * @brief Defines initialize node runtime information pass
+ * @file init_node_info.hpp
+ */
+
+#include <vector>
+#include <memory>
+
+#include <ngraph/pass/graph_rewrite.hpp>
+
+/**
+ * @brief ngraph namespace
+ */
+namespace ngraph {
+
+/**
+ * @brief ngraph::pass namespace
+ */
+namespace pass {
+
+class NGRAPH_API FixRtInfo;
+
+}  // namespace pass
+}  // namespace ngraph
+
+/**
+ * @ingroup ie_transformation_common_api
+ * @brief FixRtInfo transformation helps to set fix info attributes in a single place.
+ *        User can pass runtime attribute using various wais.
```

Comment:
`ways`

Reply:
CC @GlebKazantaev

Reply:
@apankratovantonp I don't think that we need dedicated pass for rt info fix as it duplicates graph traversals inside InitNodeInfo. Could you please move all this logic from fix_rt_info.hpp to init_node_info.cpp?

Reply:
can be done in a separate PR.

Reply:
This pass is not the same as the init node info! It does not change rt info if it is empty!